src/runner/bootstrapper.ts

```diff
@@ -93,6 +94,7 @@ export default class Bootstrapper {
     public disableMultipleWindows: boolean;
     public compilerOptions?: CompilerOptions;
     public browserInitTimeout?: number;
+    public hooks?: HooksValue;
```

Comment:
`public globalHooks?: GlobalHooks`

Reply:
This is not necessary because it's the user's property defined in the config file. All fields in the config file are global by default. Also, it will be more difficult to remember a longer name, and the chance to make mistakes will be higher.